tests/c/unmapped_setjmp.c

```diff
@@ -5,8 +5,7 @@
 //     jit-state: start-tracing
 //     set jump point
 //     jumped!
-//     jit-state: stop-tracing
-//     jit-state: trace-compilation-aborted
+//     jit-state: stop-tracing-aborted
```

Comment:
"stop tracing aborted" is confusing to me.

It sounds like we were unable to stop tracing, and are therefore still tracing.

Unless there's a good reason to, I'd keep the old behavior.

Reply:
There is a good reason: both the old message *and* the new message are wrong! "Trace compilation" wasn't aborted previously: mapping went wrong (not compilation). However, at the moment we can't tease that error out separately. So I don't like the new message, but it's equivalent incorrect to the old message, so my dislike remains, but it has neither grown or shrink ;)

Reply:
agreed (perhaps with `start-trace-compilation`/`stop-trace-compilation`/`trace-compilation-aborted(mapfail)` for consistency with the `start`/`stop` scheme we have in place.)

If you plan to change this name very soon, I'm OK with this in the interim.

Reply:
I do plan to do so soon(ish), assuming you're happy with the overall direction of this PR.